src/antq/diff/java.clj

```diff
@@ -82,10 +82,12 @@
     (cond
       (str/starts-with? url "https://github.com/")
       (let [tags (u.git/tags-by-ls-remote url)
-            current (first (filter #(str/includes? % (:version dep)) tags))
-            latest (or (first (filter #(str/includes? % (:latest-version dep)) tags))
-                       ;; If there isn't a tag for latest version
-                       "head")]
+            version-dep (:version dep)
+            latest-version-dep (:latest-version dep)
+            current (and version-dep (first (filter #(str/includes? % version-dep) tags)))
+            latest (and latest-version-dep (or (first (filter #(str/includes? % latest-version-dep) tags))
+                                               ;; If there isn't a tag for latest version
+                                               "head"))]
```

Comment:
Like src/antq/diff/github_tag.clj, it would be better to do nil check at the beginning.

Reply:
Done